repl/src/main/resources/fake_shell.py

```diff
@@ -403,6 +406,9 @@ def magic_table(name):
         cols = []
         data.append(cols)
 
+        if os.environ.get("LIVY_TEST") != "true" and isinstance(row, pyspark.sql.types.Row):
```

Comment:
I think we can do the type checking without importing the class by doing string comparsion on row.__class__.__name__.

An alternative is, we can just check does asDict() present in the row object. If it does, then call it.